pkg/service/controller.go

```diff
@@ -228,8 +230,32 @@ func (c *ControllerService) ListSnapshots(context.Context, *csi.ListSnapshotsReq
 }
 
 //ControllerExpandVolume
-func (c *ControllerService) ControllerExpandVolume(context.Context, *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "")
+func (c *ControllerService) ControllerExpandVolume(_ context.Context, req *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
+	klog.Infof("Expanding volume %v to %v bytes.", req.VolumeId, req.CapacityRange.RequiredBytes)
+
+	conn, err := c.ovirtClient.GetConnection()
+	if err != nil {
+                msg := fmt.Sprintf("Failed to get ovirt client connection")
+		klog.Errorf(msg)
+		return nil, status.Error(codes.Unavailable, msg)
+	}
+
+	// find diskAttachment and diskAttachmentService by DiskId
+	diskAttachment, diskAttachmentService, _ := diskAttachmentByDisk(conn, req.VolumeId)
+	if diskAttachment == nil || diskAttachmentService == nil {
+                msg := fmt.Sprintf("Unable to find disk attachment for volume %s.", req.VolumeId)
+		klog.Errorf(msg)
+		return nil, status.Error(codes.NotFound, msg)
+	}
+
```

Comment:
I think that we should add a check here for the disk size and return `codes.OK` if the size is already larger than or equal  to req.CapacityRange.RequiredBytes, according to the CSI docs[1]
```
This operation MUST be idempotent. If a volume corresponding to the specified volume ID is already larger than or equal to the target capacity of the expansion request, the plugin SHOULD reply 0 OK.
```

[1] https://github.com/container-storage-interface/spec/blob/master/spec.md#controllerexpandvolume